build.gradle

```diff
@@ -20,7 +22,7 @@ dependencies {
 }
 
 ext {
-    os = System.getProperty("os.name").toLowerCase(Locale.ENGLISH)
+    os = getOsName()
     platform = System.getProperty("platform") ?: os
```

Comment:
If I understand correctly, `os` is used only as the default value for `platform`. Thus, to make things simpler, maybe replace these two lines with just a single `platform = getPlatform()` and replace `getOsName()` with `getPlatform()`?

Reply:
os also used in https://github.com/roc-project/roc-java/blob/master/roc_jni/build.gradle#L10